CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ option(UR_USE_ASAN "enable AddressSanitizer" OFF)
 option(UR_USE_UBSAN "enable UndefinedBehaviorSanitizer" OFF)
 option(UR_USE_MSAN "enable MemorySanitizer" OFF)
 option(UR_USE_TSAN "enable ThreadSanitizer" OFF)
-option(UR_ENABLE_TRACING "enable api tracing through xpti" OFF)
+option(UR_ENABLE_TRACING "enable api tracing through xpti" ON)
```

Comment:
intentional?

Reply:
No, I had it turned on for testing, left it by mistake.